disnake/ext/commands/params.py

```diff
@@ -333,6 +336,23 @@ class LargeInt(int):
 _VERIFY_TYPES: Final[FrozenSet[OptionType]] = frozenset((OptionType.user, OptionType.mentionable))
 
 
+class _PrivateEnum(enum.Enum):
+    # An enum to generate sentinel values for Annotated
+    PRIVATE = enum.auto()
```

Comment:
what exactly is this used for? It doesn't seem like its used at runtime

Reply:
The sentinel value is only relevant for runtime checking, in this case it really doesn't matter what metadata you attach to the `Annotated`, as it's not used anyways.